### DISCUSSION TITLE
build (optional) translations with visual studio

### DISCUSSION THREAD

**Discussion by JohnnyR8** [diasurgical/DevilutionX#7446] [Q&A] [answered]
when building with Visual studio i only get the english translation.

optional building devilutionX.mpq File is possible, by installing smpq and adding it's path to system's PATH environment variable.

but i found no option for translations... how is this done?

when building with ubuntu there is this (optional) possibility:

build the translations:
sudo apt-get install gettext poedit

**Comment by AJenbo** [comment]
use vcpkg (package manager ud with Visual Studio) to install gettext

**Reply by tsunamistate** [reply]
To be more specific:
1) Install `vcpkg`: https://learn.microsoft.com/en-us/vcpkg/get_started/get-started-vs?pivots=shell-powershell
2) Install `gettext` with binaries: `vcpkg install gettext[tools]:x64-windows`
3) Add `{vcpkg root}\installed\x64-windows\tools\gettext\bin` to PATH: https://www.eukhost.com/kb/how-to-add-to-the-path-on-windows-10-and-windows-11/

**Reply by JohnnyR8** [reply, marked as answer]
thank you guys :-)
